zeebe/gateway-grpc/src/main/java/io/camunda/zeebe/gateway/interceptors/impl/IdentityInterceptor.java

```diff
@@ -97,24 +113,32 @@ public <ReqT, RespT> ServerCall.Listener<ReqT> interceptCall(
 
     try {
       final List<String> authorizedTenants =
-          identity.tenants().forToken(token).stream().map(Tenant::getTenantId).toList();
+          tenantService.getTenantsForToken(token).stream().map(Tenant::getTenantId).toList();
       final var context = InterceptorUtil.setAuthorizedTenants(authorizedTenants);
       return Contexts.interceptCall(context, call, headers, next);
 
-    } catch (final RuntimeException e) {
-      LOGGER.debug(
-          "Denying call {} as the authorized tenants could not be retrieved from Identity. Error message: {}",
-          methodDescriptor.getFullMethodName(),
-          e.getMessage());
-      return deny(
-          call,
-          Status.UNAUTHENTICATED
-              .augmentDescription(
-                  "Expected Identity to provide authorized tenants, see cause for details")
-              .withCause(e));
+    } catch (final RejectedExecutionException ree) {
```

Comment:
:wrench: I would propose using a specific exception type if only to avoid a possible `RejectedExcutionException` coming from the Identity SDK itself, which also uses some stuff internally via the HttpClient.